services/wordpress/wordpress-downloads.service.js

```diff
@@ -52,14 +53,28 @@ function DownloadsForExtensionType(extensionType) {
       pattern: ':interval(dd|dw|dm|dy|dt)/:slug',
     }
 
-    static examples = [
-      {
-        title: `WordPress ${capt} Downloads`,
-        namedParams: { interval: 'dm', slug: exampleSlug },
-        staticPreview: this.render({ interval: 'dm', downloads: 200000 }),
-        documentation,
-      },
-    ]
+    static get openApi() {
+      const key = `/wordpress/${extensionType}/{interval}/{slug}`
+      const route = {}
+      route[key] = {
```

Comment:
This isn't a really common pattern, but we do use it in a handful of places.

With most of the wordpress services, we define a function that returns a class, and then call that function with `'plugin'` and `'theme'` to create two variants: One for wordpress plugins, one for wordpress themes. This reduces duplication, but we can't define `openApi` as a static property because the route key can't be a variable or a template string.

```js
static openApi = {
  `/${someVar}/{param}`: //some object
}
```

is not legal javascript.

In this situation, we can use a `static get` function.